digits/model/tasks/caffe_train.py

```diff
@@ -39,6 +39,50 @@
 CAFFE_DEPLOY_FILE = 'deploy.prototxt'
 
 @subclass
+class DigitsTransformer(caffe.io.Transformer):
```

Comment:
Oh sorry - I didn't read carefully.

With this approach, it doesn't look like you're restoring the original values of `self.inputs` and `self.mean`. What's going to happen to the mean if it keeps getting resized on every call to `preprocess`? Or what happens when you want to use the same transformer to do a non-resizing transformation, then use the same object to do a resizing transformation? The image size will be dependent on the last used configuration instead of how the transformer was originally set up.